workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/TicketCard.tsx

```diff
@@ -0,0 +1,164 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import ClayPanel from '@clayui/panel';
+import {useDraggable} from '@dnd-kit/core';
+import {CSS} from '@dnd-kit/utilities';
+import {useState} from 'react';
+import {QueryClient, useMutation, useQueryClient} from 'react-query';
+
+import {Liferay} from '../services/liferay';
+import {assignTicketToMe} from '../services/tickets';
+import {Ticket} from '../types';
+
+const TicketCard: React.FC<{ticket: Ticket}> = ({ticket}) => {
+	const [isLoading, setIsLoading] = useState(false);
+	const [isPanelExpanded, setIsPanelExpanded] = useState(false);
+
+	const queryClient: QueryClient = useQueryClient();
+
+	const {
+		attributes,
+		isDragging,
+		listeners,
+		setNodeRef,
+		transform,
+	} = useDraggable({data: ticket, id: ticket.id + '_draggable'});
+
+	const style: React.CSSProperties = {
+		position: 'relative',
+		transform: CSS.Translate.toString(transform),
+		zIndex: isDragging ? 150 : 1,
+	};
+
+	const assignToMeMutation = useMutation({
+		mutationFn: async (ticket: Ticket) => {
+			setIsLoading(true);
+
+			return await assignTicketToMe(ticket);
+		},
+		onError: (error: Error) => {
+			setIsLoading(false);
+
+			Liferay.Util.openToast({
+				message: error.message,
+				title: 'Request Failed',
+				type: 'danger',
+			});
+		},
+		onSuccess: () => {
+			setIsLoading(false);
+
+			queryClient.invalidateQueries();
+
+			Liferay.Util.openToast({
+				message: 'The ticket was successfully assigned to you!',
+				type: 'success',
+			});
+		},
+	});
+
+	return (
+		<div
+			{...attributes}
+			className={`border border-neutral-2 card mb-4 py-2 shadow-none ${
```

Comment:
If you're setting the border color, the background, and removing the box-shadow, then the `card` class isn't doing anything. You should be able remove both the card and shadow-none classes.